stack.yaml

```diff
@@ -3,6 +3,7 @@ resolver: lts-12.9
 packages:
   - co-log
   - co-log-core
+  - co-log-benchmark
```

Comment:
You also need `co-log-benchmark` to `.travis.yml` here:

https://github.com/kowainik/co-log/blob/76964369c38152b2ad10bd3c452e0e7a0a90dfb9/.travis.yml#L79-L80